src/popup/router/components/Modals/AuctionsHelp.vue

```diff
@@ -0,0 +1,23 @@
+<template>
```

Comment:
There would be nice to have one modal covering all the help cases to not add each time a new one.